scripts/create_metadata

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# OUT_DIR is where the file gets written.
+# The main purpose of this is to make testing easier.
+OUT_DIR="${OUT_DIR:-.}"
+
+# Check if DEV_TAGS is unset, if not, set it to empty.
+if [ -z "${DEV_TAGS:-}" ]; then
+    DEV_TAGS=""
+fi
+
+# SPLIT splits a string by whitespace and removes any empty matches.
+SPLIT='split("[[:space:]]+"; "gn") | map(select(length > 0))'
+
+jq --null-input --sort-keys \
+	--arg tags "${TAGS}" \
+    --arg dev_tags "${DEV_TAGS}" \
+    '{
+	tags: $tags | '"$SPLIT"',
+        dev_tags: $dev_tags | '"$SPLIT"'
+    }' > "${OUT_DIR}/docker_tag_list_${TARGET}.json"
```

Comment:
Because the split code got big, I put it inside `SPLIT` to make this more readable.

I also changed `>>` to `>` so that when running the test multiple times, we don't keep appending to the same file. This also makes it idempotent in real life, always writing the entire file, so easier to reason about.